bundles/storage/org.eclipse.smarthome.storage.json/src/main/java/org/eclipse/smarthome/storage/json/JsonStorage.java

```diff
@@ -203,10 +208,10 @@ private T deserialize(Map<String, Object> jsonValue) {
 
             FileReader reader = new FileReader(inputFile);
             Map<String, Map<String, Object>> type = new HashMap<String, Map<String, Object>>();
```

Comment:
`type` is only created to retrieve the Class from it later. Can´t we just use `HashMap.class` in line 211? Wouldnt the generics be erased from the type during runtime anyway?